packages/client/components/TaskEditor/CommentEditor.tsx

```diff
@@ -164,6 +166,15 @@ const CommentEditor = (props: Props) => {
         }
       }
     }
+    const links = linkify.match(text)
```

Comment:
Initially I included this code in `useLinks` to keep things DRY but as it's only being repeated twice and as we need the above `onPasteText` code with the `ref` in this file, I felt it'd be best to keep it in the same function